azext_iot/tests/test_iot_central_unit.py

```diff
@@ -216,7 +219,12 @@ def test_should_return_updated_properties(
         twin = DeviceTwin(raw_twin)
         twin_next = DeviceTwin(raw_twin)
         twin_next.reported_property.version = twin.reported_property.version + 1
-        result = compare_properties(twin_next.reported_property, twin.reported_property)
+        monitor = PropertyMonitor(
+            cmd=None, app_id=app_id, device_id=device_id, central_dns_suffix=None,
+        )
+        result = monitor._compare_properties(
+            twin_next.reported_property, twin.reported_property
+        )
         assert len(result) == 3
         assert len(result["$iotin:urn_azureiot_Client_SDKInformation"]) == 3
         assert result["$iotin:urn_azureiot_Client_SDKInformation"]["language"]
```

Comment:
nit: it would be nice if these asserts tested for something more than just existence

Reply:
wont fix